node-mapnik/scripts/test

```diff
@@ -27,13 +27,13 @@ PGUSER=postgres psql -d template_postgis -c "GRANT ALL ON spatial_ref_sys TO PUB
 
 # Link previously built node-mapnik
 cd node-mapnik
-npm link
+sudo npm link
```

Comment:
Then what about using:

```
NPM_CONFIG_PREFIX=~/.npm-global npm link
```

And the same later on when you need to link it as a dependency.